tests/src/performance/Scenario/JitBench/Benchmarks/MLBenchmark.cs

```diff
@@ -75,10 +75,10 @@ public MLBenchmark(string name) : base(name)
             string word2VecNetPublishDir = GetWord2VecNetPublishDirectory(dotNetInstall, outputDir, tfm);
 
             // Download the corpus of text. This is a zip file that contains a text file of 100M of text from Wikipedia
-            var url = "http://mattmahoney.net/dc/text8.zip";
+            var url = "https://perfbenchmarkstorage.blob.core.windows.net/corpus/Corpus10.zip";
```

Comment:
great to see this hosted by us!

btw @adiaaida is there a way to use this ML tool and save its state after the learning phase? I just wonder if there is a way to do less for setup

Reply:
There is, we could upload to blob storage the vector.bin file that is generated, but one of the things they test in the ML space is training time, so we probably want to keep the training as part of the scenario (in particular, when you consider the cost of training if you're training in the cloud, for example).

Reply:
Oh, it does happen as part of setup. This is just pulling down the text file that will be ready by the benchmark. The benchmark itself trains based on the text file, then does a few searches.

Reply:
I am dumb, sorry for a stupid question ;p